src/lib/containers/download_list/DownloadConfirmation.tsx

```diff
@@ -38,6 +38,7 @@ export type DownloadConfirmationProps = {
   updateParentState?: <K extends keyof QueryWrapperState>(
     param: Pick<QueryWrapperState, K>,
   ) => void
+  onExportTable?: () => void
```

Comment:
Just curious, is this the same as using Function as a type?

Reply:
yes, as far as I know!